courses/machine_learning/deepdive2/how_google_does_ml/solutions/analyze_with_bigquery_solution.ipynb

```diff
@@ -37,7 +37,7 @@
   },
```

Comment:
<p>We should probably end the notebook by creating the ml dataset from the dedup data as a BQ table so that we can run BQML on it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>The deduplicated data already exists in the "all_sessions" table.    I added a note to what to do if this is not the case and the basic 'create table as select' syntax.</p>